### ISSUE TITLE
Django 3.1 emits warnings about DEFAULT_HASHING_ALGORITHM

### ISSUE THREAD

**Issue by brianhelba** [jazzband/django-configurations#263]
This is likely due to the fact that `DEFAULT_HASHING_ALGORITHM` is considered an explicitly-set setting, as it's included in the settings copied from `global_settings` to the local settings module.

**Comment by blueyed** [comment]
Also affects `DEFAULT_AUTO_FIELD`, where `settings.is_overridden('DEFAULT_AUTO_FIELD')` is used in https://github.com/django/django/blob/d29a9ed5043bd24b7c074b886125159e67ab8eba/django/db/models/base.py#L1308, which is `True`, although not overridden / defined explicitly.
Ref: https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys (models.W042)

**Comment by brianhelba** [comment]
@blueyed Good point about `DEFAULT_AUTO_FIELD`. I've reported this as #286 and fixed it in #287.
